File: dev-superior-algoritmo-e-logica-de-programacao/java/src/exercicios/ParImpar.java
package exercicios;

import java.util.Scanner;

public class ParImpar {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		int N, num;

	    System.out.print("Quantos numeros voce vai digitar? ");
	    N = sc.nextInt();

		for (int i = 0; i < N; i++) {
	        System.out.print("Digite um numero: ");
			num = sc.nextInt();

			if (num == 0) {
	            System.out.println("NULO");
			} else {
				if (num % 2 == 0) {
	            	System.out.print("PAR");
				} else {
	            	System.out.print("IMPAR");
	            }

				if (num > 0) {
	            	System.out.println(" POSITIVO");
				} else {
	            	System.out.println(" NEGATIVO");
	            }
	        }
	    }

		sc.close();
	}
}

File: nelio-alves-algoritmo-e-logica-de-programacao/c/exercicios/par_impar.c
#include <stdio.h>

int main() {
    int N, num;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &N);

    for (int i = 0; i < N; i++) {
        printf("Digite um numero: ");
        scanf("%d", &num);

        if (num == 0) {
            printf("NULO\n");
        } else {
            if (num % 2 == 0) {
                printf("PAR");
            } else {
                printf("IMPAR");
            }

            if (num > 0) {
                printf(" POSITIVO\n");
            } else {
                printf(" NEGATIVO\n");
            }
        }
    }

    return 0;
}
